core/dbt/adapters/sql/impl.py

```diff
@@ -63,35 +65,35 @@ def add_query(
                                           abridge_sql_log)
 
     @classmethod
-    def convert_text_type(cls, agate_table, col_idx):
+    def convert_text_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "text"
 
     @classmethod
-    def convert_number_type(cls, agate_table, col_idx):
+    def convert_number_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         decimals = agate_table.aggregate(agate.MaxPrecision(col_idx))
         return "float8" if decimals else "integer"
 
     @classmethod
-    def convert_boolean_type(cls, agate_table, col_idx):
+    def convert_boolean_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "boolean"
 
     @classmethod
-    def convert_datetime_type(cls, agate_table, col_idx):
+    def convert_datetime_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "timestamp without time zone"
 
     @classmethod
-    def convert_date_type(cls, agate_table, col_idx):
+    def convert_date_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "date"
 
     @classmethod
-    def convert_time_type(cls, agate_table, col_idx):
+    def convert_time_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "time"
 
     @classmethod
-    def is_cancelable(cls):
+    def is_cancelable(cls) -> bool:
         return True
 
-    def expand_column_types(self, goal, current):
+    def expand_column_types(self, goal, current, model_name: Optional[str] = None):
```

Comment:
You've added a bunch of `model_name: Optional[str] = None` parameters here - unless I'm missing something significant, these methods haven't taken that as an argument since 0.14.0.

Reply:
You're right. my IDEA complaints for an unknown reason:
![image](https://user-images.githubusercontent.com/1134248/70264678-3b447a00-1799-11ea-9d25-10277a507558.png)

Reply:
Do you have a downlevel version (`0.13.1` or less) checked out or cached somewhere that it's seeing? That's all I can imagine causing that problem.

Reply:
Ahh, you're right!